packages/cyberstorm/src/components/PaginationButton.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { MouseEventHandler, ReactNode } from "react";
+import styles from "./componentStyles/PaginationButton.module.css";
+
+export interface PaginationButtonProps {
+  isSelected?: boolean;
+  ariaCurrent?: boolean;
+  ariaLabel?: string;
+  label?: string;
+  leftIcon?: ReactNode;
+  rightIcon?: ReactNode;
+  colorScheme?: "default";
+  onClick?: MouseEventHandler<HTMLButtonElement>;
+}
+
+export const PaginationButton: React.FC<PaginationButtonProps> = (props) => {
+  const {
+    ariaCurrent,
+    ariaLabel,
+    label,
+    leftIcon,
+    rightIcon,
+    colorScheme,
+    onClick,
+    isSelected,
+  } = props;
+  return (
+    <button
```

Comment:
IIRC it was mentioned in the weekly that this should maybe be a `<a>` element instead. Although I guess it depends on what clicking the component actually does? I guess it's quite safe bet to say it only navigates (i.e. it should be `<a>`), but whoever uses the component might make it also change the state or whatever (i.e. it should be <button>). We have no control over that.

TL;DR: I've no strong opinions about this issue.

Reply:
For reference, here's [my original comment about the this](https://github.com/thunderstore-io/thunderstore-ui/pull/604#discussion_r1081837678)

Reply:
Pointed it out on original comment but this should be corrected now 👍